pvlib/tests/iotools/test_solcast.py

```diff
@@ -0,0 +1,215 @@
+import pandas as pd
+from pvlib.iotools.solcast import (

```

Comment:
```suggestion
from pvlib.iotools import (
```
I would import it as a user would